src/streaming/rules/ThroughputHistory.js

```diff
@@ -103,6 +119,19 @@ function ThroughputHistory(config) {
         if (latencyDict[mediaType].length > MAX_MEASUREMENTS_TO_KEEP) {
             latencyDict[mediaType].shift();
         }
+
+        updateEwmaEstimate(ewmaThroughputDict[mediaType], throughput, 0.001 * downloadTimeInMilliseconds, ewmaHalfLife.throughputHalfLife);
+        updateEwmaEstimate(ewmaLatencyDict[mediaType], latencyTimeInMilliseconds, 1, ewmaHalfLife.latencyHalfLife);
+    }
+
+    function updateEwmaEstimate(ewmaObj, value, weight, halfLife) {
+        const fastAlpha = Math.pow(0.5, weight / halfLife.fast);
+        ewmaObj.fastEstimate = (1 - fastAlpha) * value + fastAlpha * ewmaObj.fastEstimate;
```

Comment:
I was compensating for startup in getAverageEwma() by doing `fastEstimate = ewmaObj.fastEstimate / (1 - Math.pow(0.5, ewmaObj.totalWeight / halfLife.fast))`. That also has the benefit of not giving the first measurement more weight than the next few measurements during startup. But I need to at least put some clear documentation here because the startup compensation code is rather hidden.